modules/localization/msf/local_pyramid_map/base_map/base_map_node_config.h

```diff
@@ -15,12 +15,14 @@
  *****************************************************************************/
 #pragma once
 
-#include "modules/localization/msf/local_map/base_map/base_map_config.h"
-#include "modules/localization/msf/local_map/base_map/base_map_node_index.h"
+#include <memory>
```

Comment:
separate the standard and project includes

Reply:
fixed